test/core/origin_core/get_accumulate_gas_target.js

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Test: get_accumulate_gas_target.js
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+const web3 = require('../../test_lib/web3.js');
+const BN = require('bn.js');
+const OriginCore = artifacts.require('OriginCore');
+
+contract('OriginCore.getAccumulateGasTarget()', async (accounts) => {
+
+  /**
+   * TODO:
+   * @dev while writing this test commitMetaBlock function is not
+   *      implemented, So once its updated the tests should be updated to use
+   *      commitMetaBlock function. Currently this tests are using the genesis
+   *      meta block for testing
+   */
+
+  let originCore,
+    maxAccumulateGasLimit,
+    gas,
+    auxiliaryCoreIdentifier,
+    transactionRoot,
+    ost,
+    minimumWeight
+  ;
+
+  /** Deploys the origin core contract */
+  async function deployOriginCore(){
+    originCore = await OriginCore.new(
+      auxiliaryCoreIdentifier,
+      ost,
+      gas,
+      transactionRoot,
+      minimumWeight,
+      maxAccumulateGasLimit
```

Comment:
For multi-line arguments or parameters it is best practice to also add a trailing to the last line.
```suggestion
      maxAccumulateGasLimit,
```